tools/clang/test/CodeGenSPIRV/empty_struct.hlsl

```diff
@@ -0,0 +1,30 @@
+// RUN: %dxc -T ps_6_0 -E main -fcgl  %s -spirv | FileCheck %s -check-prefix=RELAXED
+// RUN: %dxc -T ps_6_0 -E main -fcgl  %s -spirv -fvk-use-gl-layout | FileCheck %s -check-prefix=RELAXED
+// RUN: %dxc -T ps_6_0 -E main -fcgl  %s -spirv -fvk-use-scalar-layout | FileCheck %s -check-prefix=SCALAR
+
+struct empty_struct { };
+
+
+// RELAXED: OpMemberDecorate %type_cb 0 Offset 0
```

Comment:
Shall the capabilities be checked?
Also, do you know if the capability trimming pass handles this correctly? (as in don't remove the mandatory capacity if one alignment requires it?)
(IIRC related capabilities are not trimmed, but in case)

Reply:
I was referring to the 2 mentioned capabilities. DXC don't generate those, but SPIRV-Tools can handle the 16bit version. I tried your PR on BigWheels with some empty structs, 16bit types etc to force the use of those capabilities, and seems like no layer complained.
Thanks!